### ISSUE TITLE
SELECT(space) as first command causes an AttributeError

### ISSUE THREAD

**Issue by LiraNuna** [dbcli/mycli#299]
Using version 1.7.1 and mysql 5.7.11 (unknown if related), when typing `SELECT ` (notice the trailing space) the following exception raises:

```
Traceback (most recent call last):
  File "/usr/lib/python3.4/threading.py", line 920, in _bootstrap_inner
    self.run()       
  File "/usr/lib/python3.4/threading.py", line 868, in run
    self._target(*self._args, **self._kwargs)
  File "/home/patreon/patreon_py/venv/lib/python3.4/site-packages/prompt_toolkit/interface.py", line 835, in run
    completions = list(buffer.completer.get_completions(document, complete_event))
  File "/home/patreon/patreon_py/venv/lib/python3.4/site-packages/mycli/sqlcompleter.py", line 255, in get_completions                                                                                                       
    suggestions = suggest_type(document.text, document.text_before_cursor)
  File "/home/patreon/patreon_py/venv/lib/python3.4/site-packages/mycli/packages/completion_engine.py", line 85, in suggest_type
    full_text, identifier)
  File "/home/patreon/patreon_py/venv/lib/python3.4/site-packages/mycli/packages/completion_engine.py", line 134, in suggest_based_on_last_token
    token_v = token.value.lower()
AttributeError: 'tuple' object has no attribute 'value'
```

This does not happen after a `Ctrl+C` (which goes back to the prompt) and doing the same.

**Comment by dbolser-ebi** [comment]
Just spotted this too 1.7.1 on both

- mysql

  Ver 14.14 Distrib 5.6.26, for Linux (x86_64) using  EditLine wrapper.

- mysql

  Ver 14.14 Distrib 5.1.73, for redhat-linux-gnu (x86_64) using readline 5.1

Note that although you don't get this message again, autocomplete is now borken (nothing pops up, tab doesn't offer suggestions, etc.).

**Comment by dbolser-ebi** [comment]
How do I pip install 1.6?

**Comment by dbolser-ebi** [comment]
Working again like this:
```
pip freeze
PyMySQL==0.6.2
Pygments==2.0
argparse==1.4.0
click==4.1
configobj==5.0.6
mycli==1.5.2
prompt-toolkit==0.46
pycrypto==2.6.1
six==1.9.0
sqlparse==0.1.16
wcwidth==0.1.7
```

BTW, I'm also on python 2.6.6 and seeing 'syntax errors' during version munging. Should I be on 2.7 for mycli (and it's deps)?


Thanks,

**Comment by amjith** [comment]
This is now fixed in version 1.8.0. Please upgrade mycli: 

```
pip install -U mycli
```

**Comment by dbolser-ebi** [comment]
Thanks.
